src/transformers/configuration_utils.py

```diff
@@ -85,7 +85,9 @@ class PretrainedConfig(PushToHubMixin):
         add_cross_attention (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Whether cross-attention layers should be added to the model. Note, this option is only relevant for models
             that can be used as decoder models within the `:class:~transformers.EncoderDecoderModel` class, which
-            consists of all models in ``AUTO_MODELS_FOR_CAUSAL_LM``.
```

Comment:
Actually I think we can leave this one here as it was :-) We usually only use `PyTorch`s classes as a reference in tokenizer and configuration files. We kinda assume that every functionality that exists has to exist at least in PyTorch. Sorry for only noting this now!